linode/manager
# feat: Add Metadata region capability

## Description 📝
Not every region supports Metadata. This PR adds an additional check for region capability before showing the Add User Data Accordion.

## How to test 🧪
- Turn on the MSW and go to the Linode Create page
- Check the Distributions, Images, Backups, and Clone Linode flow
    - Select the metadata-test image and a region. The Add User Data accordion should only display if the selected region supports Metadata (currently, that is only London, UK)
